tests/ui-cargo/cargo_rust_version/fail_file_attr/src/main.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(custom_inner_attributes)]
+#![clippy::msrv = "1.58.0"]
```

Comment:
So, this doesn't produce a warning. That is not optimal, but I wouldn't try to address this in this PR.

Can you add a `FIXME`, that this should produce some kind of warning, that it overrides the value from one of the toml files?

Reply:
Maybe abandon that attribute in the long run? ;)